RetailSuiteWarehousing/Integration/02_ConfigureWarehouses.md

```diff
@@ -0,0 +1,118 @@
+[!!Configure the warehouse groups](./01_ConfigureWarehouseGroups.md)
+[!!Check the posting history](../Operation/03_CheckPostingHistory.md)
+[!!User interface Warehouse](../UserInterface/03a_Warehouse.md)
+
+# Configure the warehouses
+
+A warehouse is a subordinate storage area within a warehouse group used for a specific purpose, such as sales, returns, or blocked stock. A warehouse can be assigned, or not, to a specific warehouse group. If a warehouse is not assigned to any warehouse group, it will be listed under the general *No warehouse group* in the left side bar. 
+
+Warehouses can be created, edited, and deleted. Bear in mind, though, that you cannot delete a warehouse where postings have been made in the past, as all posting information is kept in the system. For detailed information, see [Check the posting history](../Operation/03_CheckPostingHistory.md).
+
+
+
+## Create a warehouse
+
+Create one or more specific warehouses to store and manage your stock.
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+*Warehouse > Settings > Tab WAREHOUSE > Tab Warehouse master data*
+
+![Warehouse master data](../../Assets/Screenshots/RetailSuiteWarehousing/Settings/Warehouse/WarehouseMasterData/WarehouseMasterData.png "[Warehouse master data]")
+
+1. Click the [NEW WAREHOUSE] button.   
+    The *Basic data* sub-tab is displayed on the right side.
+
+     ![Warehouse basic data](../../Assets/Screenshots/RetailSuiteWarehousing/Settings/Warehouse/WarehouseMasterData/WarehouseBasicData.png "[Warehouse basic data]")
+
+2. Enter a number for the warehouse in the *Warehouse* field. This is the warehouse ID.
+
+3. If desired, select the warehouse group in the *Warehouse group* drop-down list.  
+    
+    > [Info] If you do not select a warehouse group, the warehouse will be included in the **No warehouse group**.
+
+4. Enter a name for the new warehouse in the *Designation* field.
+
+5. Click the *Products per shelf* drop-down list and select the **Different products per storage shelf** option.  
+        
+[comment]: <> (One product per storage shelf ignorieren; aktuell bei keinem Kunden benutzt und und Probleme beim Anlegen der Lagerfächer. Warehouse planning can be ignored, as it is no longer developed and has no effect.)
+
+6. Click the *Exclude from ordering* drop-down list and select **No** to deactivate the purchasing process.
+
+    > [Info] The *Purchasing* module can calculate the suggested amount of a product that needs to be purchased from a supplier to replenish the stock level. If this option is activated, that is, set to **Yes**, the stock amount stored in this warehouse will not be part of the calculation, that is, it will be ignored by the system.
```

Comment:
```suggestion
    > [Info] The *Purchasing* module can calculate the suggested amount of a product that needs to be purchased from a supplier to replenish the stock level. If this option is set to **Yes**, the stock amount stored in this warehouse will not be part of the calculation, that is, it will be ignored by the system.
```